.swiftlint.yml

```diff
@@ -0,0 +1,14 @@
+included:
+  - Source
+  - Tests
+disabled_rules:
+  - line_length
+identifier_name:
+  min_length: 1
+type_name:
+  excluded: T
+function_body_length: 100
+type_body_length: 500
+cyclomatic_complexity:
+  warning: 15
+  error: 25
```

Comment:
Don't we need a trailing newline here? I'm surprised that swiftlint doesn't complain :trollface:

Reply:
newline after what? the last line here? 
Neh its YAML, no need :)